actioncable/app/javascript/action_cable/connection_monitor.js

```diff
@@ -75,26 +73,31 @@ class ConnectionMonitor {
   }
 
   getPollInterval() {
-    const {min, max, multiplier} = this.constructor.pollInterval
-    const interval = multiplier * Math.log(this.reconnectAttempts + 1)
-    return Math.round(clamp(interval, min, max) * 1000)
+    const { staleThreshold, reconnectionBackoffRate } = this.constructor
+    const backoff = Math.pow(1 + reconnectionBackoffRate, Math.min(this.reconnectAttempts, 10))
+    const jitter = (this.reconnectAttempts > 0 ? reconnectionBackoffRate : 1) * Math.random()
```

Comment:
Do you think it would be clearer if we highlighted the special casing of the initial jittering by flipping the conditional?

```suggestion
    const jitter = this.reconnectAttempts <= 0 ? Math.random() : Math.random() * reconnectionBackoffRate
```

Or extracting a separate variable to explain the how the jitter is clamped?

```suggestion
const jitterRate = this.reconnectAttemps <= 0 ? 1 : reconnectionBackoffRate
const jitter = Math.random() * jitterRate
```

Reply:
@kaspth I've made those changes with some minor tweaks. :+1: